fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/LoadCommand.java

```diff
@@ -0,0 +1,438 @@
+        }
+        // build source columns, add select and insert node.
+        List<String> sinkCols = new ArrayList<>();
+        // contains the map of olap column to FileFieldNames, ColumnMappingList, ColumnsFromPath.
+        Map<String, NamedExpression> originSinkToSourceMap = new TreeMap<>(String.CASE_INSENSITIVE_ORDER);
+        OlapTable olapTable = getOlapTable(ctx, dataDesc);
+        Map<String, String> tvfProperties = getTvfProperties(dataDesc, bulkStorageDesc);
+
+        buildSinkColumns(olapTable, dataDesc, tvfProperties, sinkCols, originSinkToSourceMap);
+        // mapping sink colum to mapping expression
+        if (!dataDesc.getColumnMappingList().isEmpty()) {
+            for (Expression mappingExpr : dataDesc.getColumnMappingList()) {
+                if (!(mappingExpr instanceof EqualTo)) {
+                    continue;
+                }
+                EqualTo equalTo = (EqualTo) mappingExpr;
+                if (!(equalTo.left() instanceof UnboundSlot)) {
+                    // if not column, skip
+                    continue;
+                }
+                String mappingColumn = equalTo.left().toSql();
+                if (originSinkToSourceMap.containsKey(mappingColumn)) {
+                    Expression mappingExprAlias = equalTo.right();
+                    originSinkToSourceMap.put(mappingColumn, new UnboundAlias(mappingExprAlias, mappingColumn));
+                }
+            }
+        }
+        // build source columns
+        List<NamedExpression> selectLists = new ArrayList<>();
+
+        if (sinkCols.isEmpty()) {
+            // build 'insert into tgt_tbl select * from src_tbl'
+            selectLists.add(new UnboundStar(new ArrayList<>()));
+        } else {
+            for (String columnFromPath : dataDesc.getColumnsFromPath()) {
+                sinkCols.add(columnFromPath);
+                // columnFromPath will be parsed by BE, put columns as placeholder.
+                originSinkToSourceMap.put(columnFromPath, new UnboundSlot(columnFromPath));
+            }
+            checkAndAddSequenceCol(olapTable, dataDesc, sinkCols, originSinkToSourceMap);
+            selectLists.addAll(originSinkToSourceMap.values());
+        }
+        Set<Expression> conjuncts = buildTvfFilter(dataDesc, originSinkToSourceMap);
+        if (dataDesc.getFileFieldNames().isEmpty() && isCsvType(tvfProperties) && !conjuncts.isEmpty()) {
+            throw new VerifyException("Required property 'csv_schema' for csv file, "
+                    + "when use PRECEDING FILTER, WHERE, ORDER BY AND DELETE ON clause.");
+        }
+        LogicalPlan tvfLogicalPlan = new LogicalProject<>(selectLists,                            // add select
+                new LogicalFilter<>(conjuncts,                                                    // add filter
+                        new LogicalCheckPolicy<>(getUnboundTVFRelation(tvfProperties))));            // add relation
```

Comment:
u could use inline comment to make this comment more clear
```suggestion
        LogicalPlan tvfLogicalPlan = new LogicalProject<>(/*add select*/selectLists,
                /*add filter*/new LogicalFilter<>(conjuncts,
                        /*add relation*/new LogicalCheckPolicy<>(getUnboundTVFRelation(tvfProperties))));            
```